src/modules/I18n/i18next-scanner.config.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
   input: ['src/**/*.{js,ts,tsx,jsx}'],
   options: {
     debug: false,
-    removeUnusedKeys: true,
+    removeUnusedKeys: false,
```

Comment:
> Why change this? It seems quite positive to me that we don't maintain unused translations :)

Reply:
I think it's because there are dynamic translation keys that would be removed because of this.

Reply:
As Martin said, with the use of dynamc translation keys like `t('instances:${slug}.desc')` we had to pass `removeUnusedKeys` to `false`.